library/pom.xml

```diff
@@ -306,12 +306,24 @@
       <artifactId>janino</artifactId>
       <version>2.7.8</version>
       <scope>test</scope>
-    </dependency>    
-     <dependency>
+    </dependency>
+    <dependency>
```

Comment:
True but I was running into "NoSuchMethodError" exceptions as described at http://stackoverflow.com/questions/15833015/nosuchmethoderror-with-hamcrest-1-3-junit-4-11?rq=1 . I followed one of the workarounds suggested. If this is an issue I'll look into eliminating this explicit dependency.